ovos_plugin_common_play/ocp/gui.py

```diff
@@ -210,21 +210,32 @@ def manage_display(self, page_requested, timeout=None):
                 self.show_page(self.disambiguation_playlists_page, override_idle=timeout, override_animations=True)
             else:
                 self.show_page(self.disambiguation_playlists_page, override_idle=True, override_animations=True)
-                
+
+        if self.player.enable_app_view_timeout and page_requested == "player":
+            self.player.event_scheduler_interface.schedule_event(
+                self.timeout_app_view, 30, data=None, name="ocp_app_view_timer")
+
+    def timeout_app_view(self):
+        self.bus.emit(Message("homescreen.manager.show_active"))
+
     def unload_player_loader(self):
         self.send_event("ocp.gui.player.loader.clear")
 
     def show_home(self, app_mode=True):
         self.update_ocp_skills()
         self.clear_notification()
-        
+
         sleep(0.2)
         self.manage_display("home")
-        
+
         if app_mode:
             self.persist_home_display = True
         else:
-            self.persist_home_display = False       
+            self.persist_home_display = False
+
+        if self.player.state == PlayerState.PLAYING and self.player.enable_app_view_timeout:
+            self.player.event_scheduler_interface.schedule_event(
```

Comment:
maybe this could be an helper method, logic is duplicated above but with hardcoded 30 seconds instead of self.player.app_view_timeout